### PR TITLE
itR-15: weather api

### PR DESCRIPTION
**Pull request by MilaKolasava**
image

### CODE REVIEW

File: src/components/Weather/Weather.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <div
+          className="weather-render-block"
+          data-testid="weather-render-block"
+        >
+          <div className="weather-header" data-testid="weather-header">
+            <h2>{data.name}</h2>
+          </div>
+          <div className="weather-details" data-testid="weather-details">
+            <p>
+              <b>{intl.formatMessage({ id: "temperature" })}:</b>{" "}
```
Comment (glebpigulevsky) [inline src/components/Weather/Weather.tsx]: extra vscode space in the end.

File: src/components/Weather/Weather.tsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+type WeatherResponse = {
+  main: Main;
+  sys: Sys;
+  name: string;
+};
+
+function Weather() {
+  const [lat, setLat] = useState<number | null>(null);
+  const [long, setLong] = useState<number | null>(null);
+  const [data, setData] = useState<WeatherResponse | null>(null);
```
Comment (glebpigulevsky) [inline src/components/Weather/Weather.tsx]: please, do not use abbreviations. use more meaningful name than data

File: src/components/Weather/Weather.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Weather/Weather.tsx]: done

File: src/components/Weather/Weather.tsx (hunk 2 above)
Comment (MilaKolasava) [reply src/components/Weather/Weather.tsx]: done

File: src/components/Weather/__spec__/mockNavigatorGeolocation.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+export const mockNavigatorGeolocation = () => {
```
Comment (glebpigulevsky) [inline src/components/Weather/__spec__/mockNavigatorGeolocation.ts:1]: Do we need to keep this file?

**Review by glebpigulevsky — APPROVED** [review]

File: src/components/Weather/__spec__/mockNavigatorGeolocation.ts (hunk 3 above)
Comment (MilaKolasava) [reply src/components/Weather/__spec__/mockNavigatorGeolocation.ts:1]: I think yes, we need.

**Pull request merged**